### PR TITLE
Fix `releases-tab` count /3

### PR DESCRIPTION
**Pull request by yakov116**

### CODE REVIEW

File: source/features/releases-tab.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import {appendBefore, highlightTab, unhighlightTab} from '../helpers/dom-utils';
 const getCacheKey = (): string => `releases-count:${getRepo()!.nameWithOwner}`;
 
 async function parseCountFromDom(): Promise<number> {
-	const moreReleasesCountElement = await elementReady('.file-navigation .octicon-tag + strong');
+	const moreReleasesCountElement = await elementReady('.Layout-sidebar .BorderGrid-cell h2 a[href$="/releases"] .Counter');
```
Comment (fregante) [inline source/features/releases-tab.tsx:20]: I don’t think it needs to be so specific

Comment (yakov116) [reply source/features/releases-tab.tsx:20]: I took it from here

https://github.com/refined-github/refined-github/blob/9b60bfe4049e4f02a75d800784b9ce8784723394/source/features/clean-repo-sidebar.tsx#L10

Comment (fregante) [reply source/features/releases-tab.tsx:20]: We can fix both

Comment (yakov116) [reply source/features/releases-tab.tsx:20]: Looks like we went though a few bugs with this selector already I will make a comment code https://github.com/refined-github/refined-github/pull/4428

**Pull request merged**